static/js/hotkey.js

```diff
@@ -37,12 +39,28 @@ import * as stream_popover from "./stream_popover";
 import * as subs from "./subs";
 import * as topic_zoom from "./topic_zoom";
 import * as ui from "./ui";
+import * as unread from "./unread";
 
 function do_narrow_action(action) {
     action(message_lists.current.selected_id(), {trigger: "hotkey"});
     return true;
 }
 
+function check_unread_message() {
+    const last_user_msg = message_lists.current.get_last_message_sent_by_me();
+
+    if (last_user_msg && narrow_state.operators().length === 1) {
```

Comment:
I'm not sure this operators check makes sense; don't we just want to check if the user is narrowed at all?